cmd/tsgo/diagnostics.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+// max returns the larger of x or y
+func max(x, y int) int {
```

Comment:
This is already built into the language.

Reply:
Sorry, this is is admittedly AI slop